CHANGELOG.md

```diff
@@ -1,16 +1,45 @@
+# 1.4.9 (2023-02-14)
+
```

Comment:
Beginning with this release we are publishing installable packages for Ubuntu 22.04 (amd64, arm64) on Microsoft's [Linux package repository](https://packages.microsoft.com/docs/readme.txt).  Notes that on Ubuntu 22.04 the IoT Edge security subsystem has only been tested with openssl 3.0 and may not function properly if older versions of openssl have also been installed. If you previously installed openssl 1.1 to use an earlier version of IoT Edge then we would recommend removing both and starting fresh to avoid potential incompatibilities.

Reply:
Added.